### DISCUSSION TITLE
Casting shadows from gltf model to a plane doesn not work

### DISCUSSION THREAD

**Discussion by kevinsuwito** [hiukim/mind-ar-js#67] [Q&A]
It seems that GLTF models that are inside the targetIndex doesn't cast shadows. Is there something I miss or does MindAR disable shadow by default?

**Comment by hiukim** [comment]
I haven't tried shadows before, but I don't think MindAR did anything to disable it.

I'm guessing maybe it's the light position. The camera and the anchored objects position are controlled by the engine, but the light didn't move.

**Comment by BrunoFigueiraBarros** [comment]
[kevinsuwito](https://github.com/kevinsuwito) Were you able to solve GLTF with shadow?
